WebApp/src/actions/timetableActions.js

```diff
@@ -127,3 +129,54 @@ export function hideCourseTimetable(course) {
     payload: { course }
   };
 }
+
+export function cancelExchangeMode(){
+  return {
+    type: CANCEL_EXCHANGE_MODE
+  };
+}
+
+
+export function exchangeConfirm(blockTo) {
+  var action = {
+    type: CONFIRM_EXCHANGE_REQUEST,
+    payload: { blockTo }
+  };
+
+  return (dispatch, getState) => {
+    const { timetable } = getState();
+    var bl = timetable.blockFromExchange;
+    const body = {
+      BlockFrom:
```

Comment:
you do not have to use uppercase property names, API will be fine with normal blockFrom/CourseId/... names
also, do not wrap property names in double quotes, it is unnecessary, unless they are other than [a-zA-Z]+

Reply:
Ok  i fixed it